cinema/urls.py

```diff
@@ -1 +1,23 @@
 # write urls here
+from os.path import pathsep
```

Comment:
The import of `pathsep` from `os.path` is unnecessary as it is not used in this file. Consider removing it to clean up the code.